src/utils/alert.ts

```diff
@@ -109,84 +127,78 @@ export const sendAlerts = async ({
-                url,
-              }))
-            }
-            case 'slack': {
-              return sendSlack({
-                ...notification.data,
-                body: {
-                  url,
-                  alert: val.alert,
-                  time: new Date().toLocaleString(),
-                },
-              } as WebhookData).then(() => ({
-                notification: 'slack',
-                alert: val.alert,
-                url,
-              }))
-            }
-            case 'smtp': {
-              const transporter = createSmtpTransport(
-                notification.data as SMTPData
-              )
-              return sendSmtpMail(transporter, {
-                from: 'monika@hyperjump.tech',
-                to: (notification?.data as SMTPData)?.recipients?.join(','),
-                subject: message.subject,
-                html: message.body,
-              })
-            }
-            default:
-              return Promise.resolve({
-                notification: '',
-                alert: val.alert,
-                url,
-              })
-          }
-        })
-      )
+
+  const message = getMessageForAlert(validation.alert, url, ipAddress, status)
+  const sent = await Promise.all<any>(
+    notifications.map((notification) => {
+      switch (notification.type) {
+        case 'mailgun': {
+          return sendMailgun(
+            {
+              subject: message.subject,
+              body: message.body,
+              sender: {
+                // TODO: Change this before release
+                name: 'Monika',
+                email: 'Monika@hyperjump.tech',
+              },
+              recipients: (notification?.data as MailgunData)?.recipients?.join(
+                ','
+              ),
+            },
+            notification
+          ).then(() => ({
+            notification: 'mailgun',
+            alert: validation.alert,
+            url,
+          }))
+        }
+        case 'webhook': {
+          return sendWebhook({
+            ...notification.data,
+            body: {
+              url,
+              alert: validation.alert,
+              time: new Date().toLocaleString(),
+            },
+          } as WebhookData).then(() => ({
+            notification: 'webhook',
+            alert: validation.alert,
+            url,
+          }))
+        }
+        case 'slack': {
+          return sendSlack({
+            ...notification.data,
+            body: {
+              url,
+              alert: validation.alert,
+              time: new Date().toLocaleString(),
+            },
+          } as WebhookData).then(() => ({
+            notification: 'slack',
+            alert: validation.alert,
+            url,
+          }))
+        }
+        case 'smtp': {
+          const transporter = createSmtpTransport(notification.data as SMTPData)
+          return sendSmtpMail(transporter, {
+            from: 'http-probe@hyperjump.tech',
```

Comment:
Can we put TODO comment or read this configuration from env variable?

Reply:
Done